src/Microsoft.ML.StaticPipe/TextStaticExtensions.cs

```diff
@@ -492,15 +492,15 @@ public bool Equals(Reconciler other)
         /// <param name="ngramLength">Ngram length.</param>
         /// <param name="skipLength">Maximum number of tokens to skip when constructing an ngram.</param>
         /// <param name="allLengths">Whether to include all ngram lengths up to <paramref name="ngramLength"/> or only <paramref name="ngramLength"/>.</param>
-        /// <param name="maxNumTerms">Maximum number of ngrams to store in the dictionary.</param>
+        /// <param name="maximumTermCount">Maximum number of ngrams to store in the dictionary.</param>
```

Comment:
>ngrams [](start = 61, length = 6)

whatever you decide, should keep it consistent with the description.

Reply:
Yes, thanks!

---
In reply to: [264022515](https://github.com/dotnet/machinelearning/pull/2898#discussion_r264022515) [](ancestors = 264022515)